src/lib.rs

```diff
@@ -428,6 +428,22 @@ impl Build {
         self
     }
 
+    /// removes a flag from self.flags that was added by Build.flag()
+    ///
+    /// # example
```

Comment:
Since doc for other functions uses capitalized `Example`, I suggest doing the same here:

```suggestion
    /// # Example
```